src/syncerd/types.rs

```diff
@@ -60,7 +60,13 @@ pub struct SweepBitcoinAddress {
 #[derive(Clone, Debug, Display, StrictEncode, StrictDecode, Eq, PartialEq, Hash)]
 #[display(Debug)]
 pub struct Abort {
-    pub id: u32,
+    pub task_id_or_all_tasks: TaskIdOrAllTasks,
+}
+
+#[derive(Clone, Debug, StrictEncode, StrictDecode, Eq, PartialEq, Hash)]
+pub enum TaskIdOrAllTasks {
```

Comment:
Can we rename it to something like `TaskTarget` ;p ?

Reply:
Done in 49bd62ad92535b759936536880f457e13ead24ba